### ISSUE TITLE
environment constants?

### ISSUE THREAD

**Issue by colinsurprenant** [elastic/logstash#9349]
I noticed that a bunch on *environment* constants have been defined in the `Runner` class, namely

https://github.com/elastic/logstash/blob/a9c97d4b0eff74c493aa295d27bbad78a9a5326b/logstash-core/lib/logstash/runner.rb#L7-L9

Unless there is a specific reason for not doing so, shouldn't these be defined in the `LogStash::Environment` module?

**Comment by andrewvc** [comment]
Yes. That's a good point, they should. I'd like to defer fixing this until x-pack is open, since the weird repo state internally makes this refactor kind of funky, and this is a low priority issue.
